packages/contracts/contracts/L2/messaging/L2StandardBridge.sol

```diff
@@ -70,10 +77,90 @@ contract L2StandardBridge is IL2ERC20Bridge {
         uint256 _amount,
         uint32 _l1Gas,
         bytes calldata _data
-    ) external virtual {
+    ) external payable virtual {
         _initiateWithdrawal(_l2Token, msg.sender, _to, _amount, _l1Gas, _data);
     }
 
+    function withdrawETH() external payable {
+        _initiateETHWithdrawal(msg.sender, msg.sender, msg.value, 30000, hex"");
+    }
+
+    function withdrawETH(uint32 _l1Gas, bytes calldata _data) external payable {
```

Comment:
Delete this function and people who want to send calldata can just use `withdrawETHTo`